apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PriorityList.cs

```diff
@@ -19,7 +19,8 @@
 namespace Google.Cloud.Spanner.V1
 {
     /// <summary>
-    /// A simple sorted list that allows items to return equality for IComparable.CompareTo
+    /// A simple sorted list that sorts based on the result of CompareTo,
```

Comment:
i think I just noticed it when doing this PR.  If its not a big deal, would like to just leave it in here.  If its a big issue, I can create a separate PR.